### PR TITLE
Em 15

### PR DESCRIPTION
**Pull request by lalodiazdiaz**

### CODE REVIEW

File: src/components/Loader/Loader.module.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.loaderContainer{
```
Comment (FraGa96) [inline src/components/Loader/Loader.module.css]:
```suggestion
.loaderContainer {
```

File: src/components/Loader/Loader.module.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  display: flex;
+  width: 90%;
+  height: 20%;
+  background-color: white;
+  border-radius: 5px;
+  border: 3px solid #B2B9BA;
+  align-items: center;
+  justify-content: center;
+}
+.loaderContainer2{
```
Comment (FraGa96) [inline src/components/Loader/Loader.module.css]:
```suggestion

.loaderContainer2 {
```

File: src/components/Loader/Loader.module.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  display: flex;
+  width:100%;
+  height: 90%;
+  background-color: white;
+  border-radius: 5px;
+  border: 3px solid #B2B9BA;
+  align-items: center;
+  justify-content: center;
+}
+.loaderContainer p{
```
Comment (FraGa96) [inline src/components/Loader/Loader.module.css]:
```suggestion

.loaderContainer p {
```

File: src/components/NextAppointment/NextAppointent.jsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+				</div>
+
+			</div>
+			<div className={styles.contButtons}>
+				<button
+					className={range === RANGE.patient
+						? styles.btnVideocall
+						: styles.btnVideocalls}
+					type="button"
+				>Videollamada
```
Comment (FraGa96) [inline src/components/NextAppointment/NextAppointent.jsx]: Align the text properly
```suggestion
				>
				  Videollamada
```

File: src/components/NextAppointment/NextAppointent.jsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+				</button>
+				<button
+					className={range === RANGE.patient
+						? styles.btnCancel
+						: styles.btnsCancel}
+					disabled={isVisible}
+					onClick={handleDeleteAppointment}
+					type="button"
+				>Cancelar
```
Comment (FraGa96) [inline src/components/NextAppointment/NextAppointent.jsx]: Align the text properly
```suggestion
				>
				  Cancelar
```

File: src/components/NextAppointment/NextAppointent.jsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+function NextAppointment({ onAction, date, name, time, title, id }) {
+	const [isVisible, setIsVisible] = useState(false);
+	 const dispatch = useDispatch();
+	const local = JSON.parse(localStorage.getItem('user'));
+	const { token, range } = local.data;
+	const AuthStr = `Bearer ${token}`;
+
+	const handleDeleteAppointment = async () => {
+		const btnCancel = document.getElementById('btnCancel');
```
Comment (FraGa96) [inline src/components/NextAppointment/NextAppointent.jsx]: why are you using this reference?

File: src/components/Patient/DatesHomeworks/DatesHomeworks.module.css [hunk 7]
```diff
@@ -4,19 +4,50 @@
...
+  border-radius: 5px;
+  border: 3px solid #B2B9BA;
+  align-items: center;
+  justify-content: center;
+}
+
+.notAppointments p{
+font-size: 25px;
+color: gray;
+}
```
Comment (FraGa96) [inline src/components/Patient/DatesHomeworks/DatesHomeworks.module.css]: fix the alignment and add an empty space between p and { (this for all classes)

File: src/components/Psychologist/Dates/Dates.module.css [hunk 8]
```diff
@@ -11,13 +11,60 @@
...
+  border: 3px solid #B2B9BA;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+.loaderContainer p{
+font-size: 25px;
+color: gray;
+}
```
Comment (FraGa96) [inline src/components/Psychologist/Dates/Dates.module.css]: align properly the block and add a space between p

File: src/components/ScheduleModal/ScheduleModal.module.css [hunk 9]
```diff
@@ -4,6 +4,9 @@
   justify-content: space-around;
   align-items: center;
 }
+.contact h2{
+font-size: 25px;
+}
```
Comment (FraGa96) [inline src/components/ScheduleModal/ScheduleModal.module.css]: this is misaligned

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by FraGa96 — APPROVED** [review]

**Review by MiguelAngelSF — APPROVED** [review]

**Pull request merged**